### PR TITLE
Inspector-faq-update

### PR DESCRIPTION
**Pull request by 1212spark**
- Rename "faq-inspector" to "inspector-faq"
- Reorder Inspector Quickstart to focus on setup before providing functionality content (given that this is supposed to help people get going, not provide an overview)
- Add references in both the FAQ and Quickstart to building the tracking plan via Inspector

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206526643387953

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝 Changed routes:


| [**/faqs/inspector-faq**](https://docs-81tylwhr7.avo-dev.app/faqs/inspector-faq) | [**…/start-using-inspector**](https://docs-81tylwhr7.avo-dev.app/inspector/start-using-inspector) |
|:-:|:-:|
|<a href="https://docs-81tylwhr7.avo-dev.app/faqs/inspector-faq">Screenshot of /faqs/inspector-faq</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/eor5ExSbg8G32rJBodIZPsB7hu7RSAU0GHgF4ESvYOP1ssQIVdteU5Tlr-QEDU4ZQMAA37wS-oAzhdsWy9V-Pg.png">(view full size)</a> |<a href="https://docs-81tylwhr7.avo-dev.app/inspector/start-using-inspector">Screenshot of /inspector/start-using-inspector</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/JMrUcQvxl3ArVLf4oykjIQfnjJ20sCGtxI5w-lp2RhnVYFIzCKD9gRDhV8DM9j1y8guvbX6-LqS_OJdO8jocyQ.png">(view full size)</a> |







Commit 5bdb2e4998c0472de6e6e8bcab99474c118108a4 (https://docs-81tylwhr7.avo-dev.app).

File: pages/faqs/inspector-faq.mdx
Comment (logason) [inline pages/faqs/inspector-faq.mdx:1]: Todo: Add redirect

File: pages/faqs/inspector-faq.mdx [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Inspector FAQ
 
+### I'm new to Avo. Is Inspector a good place to start?
+
+Yes! Inspector provides you with immediate observability into your tracking plan and you can even build your tracking plan based on live data via Inspector. To get going quickly, check out our [Inspector Quickstart](/inspector/start-using-inspector)
```
Comment (logason) [inline pages/faqs/inspector-faq.mdx:5]:
```suggestion
Yes! Inspector provides you with immediate observability into your current state of tracking and you can even build your tracking plan based on live data via Inspector. To get going quickly, check out our [Inspector Quickstart](/inspector/start-using-inspector)
```

File: pages/inspector/start-using-inspector.mdx [hunk 2]
```diff
@@ -17,9 +17,9 @@ Teams spend months and years and pay for expensive data audits, only to be left
...
-- Implementation status for every analytics release in your Avo Branch, available for Development, Staging and Production environments. This gives you confidence and visibility into the status and health of the events that developers are implementing.
+- Implementation status for every analytics release in your Avo Branch, available for Development, Staging and Production environments. This gives you confidence in and visibility into the status and health of the events that developers are implementing.
 - Implementation status for the events in your entire tracking plan, so you can see which events are actually implemented and whether they’re sent correctly or not.
-- Inspector helps you migrate your tracking plan to Avo. Skip importing things you’re skeptical of. Then leverage Inspector to detect how events and properties are actually tracked, and gradually add them to your tracking plan via Avo branches.
+- The ability to seamlessly build your Avo tracking plan on top of live data, with the option to build gradually using several Avo branches or through a bulk import.
```
Comment (logason) [inline pages/inspector/start-using-inspector.mdx:22]:
```suggestion
- The ability to seamlessly build your Avo tracking plan on top of live data, with the option to build gradually or through a bulk import directly from Inspector. 
```

File: pages/inspector/start-using-inspector.mdx [hunk 3]
```diff
@@ -29,32 +29,7 @@ Inspector does not process property values or PII data, only event names, proper
...
-
-![Inspector tab](/images/workspace/nav-inspector.png)
-
-Learn more about the [Events view here](/inspector/inspector-events-view).
-
-> The production environment in Inspector is designed to handle large amounts of data and has up to 2 hours delay from receiving events to displaying them on a dashboard.
-
-> Events on dev and staging have no delay in processing and will be shown on the dashboard immediately.
-
-## How to send metadata to the Inspector?
```
Comment (logason) [inline pages/inspector/start-using-inspector.mdx:57]: I'd like to keep the product overview above the setup instructions. You don't have enough motivation to consider setups until you've learned more about the end product

File: pages/faqs/inspector-faq.mdx
Comment (logason) [reply pages/faqs/inspector-faq.mdx:1]: Done in 2d87652

**Comment by 1212spark** [comment]
@logason suggestions accepted and reordering done!

**Review by logason — APPROVED** [review]

**Pull request merged**